### ISSUE TITLE
Rename `result` to `status` in upload API response schema

### ISSUE THREAD

**Issue by daniil-berg** [innocampus/videbo#21]
For the sake of consistency, the `result` field in the `FileUploaded` response model should be renamed to `status`. This would align it with all the other storage response schemas. Right now it is the "odd one out" among the [`models`](https://github.com/innocampus/videbo/blob/e7b53e0d6668e2b47b8a9a5342e8123a98845f12/src/videbo/storage/api/models.py#L255).

https://github.com/innocampus/videbo/blob/e7b53e0d6668e2b47b8a9a5342e8123a98845f12/src/videbo/storage/api/models.py#L294-L298

I believe this would have no impact on `mod_videoservice` because the VueJS logic handling that response does not consider the `result` property anyway. @larsbonczek I am tagging you here again, so you can verify this.

**Comment by larsbonczek** [comment]
Correct, mod_videoservice does not read that field. I think you can change it.
